llama_index/llama_dataset/generator.py

```diff
@@ -67,7 +60,7 @@ def __init__(
     ) -> None:
         """Init params."""
         if service_context is None:
-            service_context = _get_default_service_context()
+            service_context = service_context or ServiceContext.from_defaults()
```

Comment:
The original default service context was setting the temperature of the llm to be zero so that question results are consistent, and the chunk size to be larger so that the parsed nodes are larger.

Feels like we should be preserving those defaults somehow

Reply:
I think at a minium we can pass in the chunk size into `from_defaults()`

Reply:
@logan-markewich 
However, an error will occur if service_context is not passed.
service_context or ServiceContext.from_defaults(chunk_size_limit=3000)
can I update the code and push for review ?

Reply:
Yea, that's what I meant there :)